src/components/button/_button-base.scss

```diff
@@ -26,6 +21,12 @@ $md-fab-padding: 16px !default;
 $md-mini-fab-size: 40px !default;
 $md-mini-fab-padding: 8px !default;
 
+// Icon Button standards
+$md-icon-button-size: 40px !default;
+$md-icon-button-border-radius: $md-fab-border-radius;
```

Comment:
Good catch. Could you just say `50%`, though, rather than re-using the fab variable? Icon buttons aren't round _because_ FABs are round, they just happen to also be round.